server/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Something has gone wrong as the node_modules are still in. Maybe do **/ or * node_modules to exclude it wherever it is found